docs/CHANGELOG.asciidoc

```diff
@@ -94,6 +94,22 @@ This can manifest as a validation error causing Kibana to not start.
 For more information, refer to {kibana-pull}190590[#190590].
 ====
 
+[discrete]
+[[known-187823]]
+.Connectors require update due to Microsoft Teams product retirement
+[%collapsible]
+====
+*Details* +
+One of the methods for configuring incoming webhooks in Microsoft Teams is being retired.
```

Comment:
I wouldn't say `One of the methods`, it was the only one available.

So maybe rather `The old method of configuring incoming webhooks in...` ?

Reply:
Thanks for the feedback! I updated via 305a011b7bd93e896b4932f53fa2d5f58fb2c904